### PR TITLE
[ADD] project_stock_request

### PR DESCRIPTION
**Pull request by patrickrwilson**
This module allows users to create stock request orders from projects and project tasks.
The stock request module is named stock_request (https://github.com/OCA/stock-logistics-warehouse)

### CODE REVIEW

**Comment by patrickrwilson** [comment]
ping @max3903

**Comment by pedrobaeza** [comment]
According to the module description, module name should be `project_stock_request`, not `project_stock`.

**Comment by patrickrwilson** [comment]
> According to the module description, module name should be `project_stock_request`, not `project_stock`.

Makes sense to me, this has been changed.

**Comment by rafaelbn** [comment]
Please, don't forget the module https://github.com/OCA/project/tree/11.0/project_task_material_stock in order to be compatible. This module was migrated here: https://github.com/OCA/project/pull/545 but for any reason author cannot finish it ( cc @HaraldPanten @ValentinVinagre )

**Comment by patrickrwilson** [comment]
> Please, don't forget the module https://github.com/OCA/project/tree/11.0/project_task_material_stock in order to be compatible. This module was migrated here: #545 but for any reason author cannot finish it ( cc @HaraldPanten @ValentinVinagre )

@rafaelbn I feel like this module is different enough than the one you mentioned above. That one seems to be used to track and consume materials on project tasks where as the project_stock_request module links to OCA's stock_request module which is used as a way to request inventory which goes through the normal warehouse processes (shipping, transfers, etc).

**Comment by rafaelbn** [comment]
Hi @patrickrwilson , thanks for explanation. I set nedd fixing because bots are :red_circle:

**Comment by patrickrwilson** [comment]
@rafaelbn perhaps you can help me with the runbot error. It's saying "You try to install module 'project_stock_request' that depends on module 'stock_request'.\nBut the latter module is not available in your system."
How do i resolve that when the module is in another repo?

**Comment by max3903** [comment]
@patrickrwilson You add the name of the other repo in the `oca_dependencies.txt` file at the root of this repo.

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 web
 timesheet
+stock-logistics-workflow
```
Comment (max3903) [inline oca_dependencies.txt]: @patrickrwilson `stock_request` is in `stock-logistics-warehouse` ;)

Comment (patrickrwilson) [reply oca_dependencies.txt]: doh, ok, updating now

**Comment by patrickrwilson** [comment]
@rafaelbn @max3903 all green now, can either of you reset the label back to needs review please?

**Comment by pedrobaeza** [comment]
Sorry, I'm just going to holidays and not directly interested on *request modules, so I'm not able to review. I just acted as OCA reviewer observing the PRs that incomes for general guidelines reviewer.

**Review by rafaelbn — APPROVED** [review]
Functionally review in runbot :+1: 

@patrickrwilson remember the OCA rule, review 3 PR per each 1 you make :smile: this makes OCA rocks!

**Review by max3903 — APPROVED** [review]

**Pull request merged**